HotelBookingTests/HotelExampleTwoStepPaymentTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using HotelBase;
+using HotelBooking;
+using NUnit.Framework;
+using Rhino.Mocks;
+
+namespace HotelBookingTests
+{
+    [TestFixture]
+    class HotelExampleTwoStepPaymentTest : MockedServices
```

Comment:
To consider:

I am not sure if this comments belongs to this class only, but IMHO your tests put my attention on it.

In your test method you have methods metion about Authorization and Capture, but your class code simple cannot know about it at all, because it is transparent (hide by PaymentService). If this class will pass some authorization param or, create operation called 'capture' than it makes sense.

Maybe it is worth to add that Authorization and Capture and well know names in Payment domains. There are not just technical names and it will be good in some tests (doesn't have to be this test class) to show that two steps booking are compose of Authorization and Capture payments

Reply:
applied